.github/workflows/build.yml

```diff
@@ -0,0 +1,70 @@
+name: 'Build'
+
+on: [push]
```

Comment:
we don't need to run this action for all builds. 
only if `plugin.yaml` was pushed.